src/common/prntbase.cpp

```diff
@@ -2144,10 +2162,11 @@ bool wxPrintPreviewBase::DrawBlankPage(wxPreviewCanvas *canvas, wxDC& dc)
         shadowOffset, paperRect.height);
 
     // Draw blank page allowing for 1-pixel border AROUND the actual paper
-    dc.SetPen(*wxBLACK_PEN);
+    dc.SetPen(wxPen(*wxBLACK, dc.FromDIP(1), wxPENSTYLE_SOLID));
```

Comment:
This is pretty minor, but I thought creating `wxPen` with the default width was supposed to do the right thing in any DPI. And I think it would make sense to make this work...

Reply:
`*wxBLACK_PEN` is a 1 pixel wide `wxPen` object that is created and cached in `wxStockGDI`.
It doesn't know the DPI of the `wxDC`.
`*wxThePenList->FindOrCreatePen(*wxBLACK, dc.FromDIP(1), wxPENSTYLE_SOLID)` could be used to cache the pen, but I don't think that is necessary in this case.
So I think that for now, creating a temporary `wxPen` whose width depends on DPI is the right thing to do.